src/main/java/frc/robot/subsystems/Claw.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.revrobotics.SparkMaxLimitSwitch;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Claw extends SubsystemBase {
+  public Claw() {
+
+    forewardLimit.enableLimitSwitch(false);
+    reverseLimit.enableLimitSwitch(false);
+
+    isClosed = forewardLimit.isPressed();
+    isOpen = reverseLimit.isPressed();
+  }
+
+  private CANSparkMax clawMotor = new CANSparkMax(8, MotorType.kBrushless);
+
+  private SparkMaxLimitSwitch forewardLimit =
+      clawMotor.getForwardLimitSwitch(SparkMaxLimitSwitch.Type.kNormallyOpen);
+  private SparkMaxLimitSwitch reverseLimit =
+      clawMotor.getReverseLimitSwitch(SparkMaxLimitSwitch.Type.kNormallyOpen);
+
+  private boolean isClosed;
+  private boolean isOpen;
+
+  public boolean queryClosed() {
+    return isClosed;
```

Comment:
don't yyou need to actually call 'forwardLimit.isPressed()' here?

I think you're going to return the same value forever (you only set the value in the Constructor, and never update it.